lib/components/ms-card/MsSummaryCardItem.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <div class="ms-summary-card-item">
+    <p class="ms-summary-card-item__label form-label">{{ $msTranslate(label) }}</p>
+    <p class="ms-summary-card-item__text subtitles-normal">{{ text }}</p>
+    <p
+      v-show="error"
+      class="form-error body"
+    >
+      {{ $msTranslate(error) }}
+    </p>
+    <div
+      v-if="secondLine"
+      class="ms-summary-card-item__text subtitles-normal"
+    >
+      {{ secondLine.text }}
```

Comment:
Should be translated.

Reply:
Nope, we pass value here not a label, could be anything coming from user input, server data...